shell/mixins/harvester-disk.js

```diff
@@ -0,0 +1,17 @@
+import { findBy } from '@shell/utils/array';
```

Comment:
Can this file be moved to pkg/harvester/mixins? Otherwise, the PR looks good.

Reply:
Done,  PTAL.  thank